joomla/joomla-cms
# Category Manager News Feeds count

This patch creates in "**Category Manager: News Feeds**" 4 new columns with the number of newsfeeds per category. It contains a helper file for com_newsfeeds that is used in com_categories to calculate the number of newsfeed items (published, unpublished, archived, and trashed) in each category.
## Test instructions:

The Category Manager: News Feeds (Components > NewsFeeds > Categories) does not display the number of News Feeds in each category.

![screen shot 2015-05-13 at 15 17 09](http://issues.joomla.org/uploads/1/081e5f625ac6870a26d10fad79da2a76.png)

Create some new newsfeeds that are unpublished, trashed & archived.
**Note** This patch **works only in combination with** #6916. Install both #6916 and this patch (the order of installation does not matter, you just need both).
After installing the patch, The Category Manager: NewsFeeds (Components > NewsFeeds > Categories) should displays 4 new columns.

![screen shot 2015-05-13 at 15 17 09](http://issues.joomla.org/uploads/1/999e79efc99c25364bd820cb4e34bbc5.png)

Please also check this patch with the Hathor admin template.
